olingo-odata2-core/src/main/java/org/apache/olingo/odata2/core/ep/consumer/XmlPropertyConsumer.java

```diff
@@ -51,15 +52,15 @@ public class XmlPropertyConsumer {
   protected static final String FALSE = "false";
```

Comment:
This class is just copied over from the new sources. It seems that changes are only in formatting and there are a few additions in newer version. If it is confirmed that there is no custom modifications in this class any more I'll remove this override and fall back to original. Otherwise please point to those where are the custom modifications removals that need to be restored.

Reply:
By restoring version `2.0.11` of those classes it's clear that they were changed only because of compilation issues that are now fixed in `2.0.13` so that this override can be removed.